group_vars/tigerdata/production.yml

```diff
@@ -0,0 +1,51 @@
+---
+postgres_host: "lib-postgres-prod3.princeton.edu"
+passenger_server_name: "tigerdata-prod.princeton.edu"
+passenger_app_env: "production"
+app_db_host: '{{ postgres_host }}'
+app_db_name: '{{ vault_tigerdata_prod_db_name }}'
+app_db_user: '{{ vault_tigerdata_prod_db_user }}'
+app_db_password: '{{ vault_tigerdata_prod_db_password }}'
+app_secret_key: '{{ vault_tigerdata_prod_secret_key }}'
+rails_app_env: "production"
+tigerdata_honeybadger_key: '{{ vault_honeybadger_key }}'
+datadog_api_key: "{{ vault_datadog_key }}"
+
+datadog_config:
+  log_enabled: true
+  apm_enabled: true
+  process_config:
+    enabled: true
+  apm_config:
+    analyzed_spans:
+      lockers-and-study-spaces|rack.request: 1
+datadog_typed_checks:
+  - type: tls
+    configuration:
+      init_config:
+      instances:
+        - server: lockers-and-study-spaces.princeton.edu
+          port: 443
+  - type: process
+    configuration:
+      init_config:
+      instances:
+        -   name: lockers-and-study-spaces
+            service: lockers-and-study-spaces
```

Comment:
If this is a service that is used by multiple applications, maybe add a comment explaining that? As I'm reading this, my first guess is that it's a mistaken copy-paste, but I could be wrong.

Reply:
Yo are right this is a copy paste error.